parsers.js

```diff
@@ -19,6 +19,8 @@ function resolveParser(type) {
       return autocomplete;
     case "array":
       return array;
+    case "sshKey":
+      return sshKey;
```

Comment:
Why not just return `string` parser here? `sshKey` parser function is not exported anyway, so it is just a wrapper for `string` parser, right? Unless the `sshKey` parser will be more complex in the future and right now this function is just a basis for future development.

Reply:
Initially the plan was to code it, but then I've realised I could just reuse string.
Good one.